sdk/deviceprovisioningservices/Azure.ResourceManager.DeviceProvisioningServices/src/Generated/Models/DeviceProvisioningServicesCertificateProperties.cs

```diff
@@ -10,45 +10,45 @@
 namespace Azure.ResourceManager.DeviceProvisioningServices.Models
 {
     /// <summary> The description of an X509 CA Certificate. </summary>
-    public partial class CertificateProperties
+    public partial class DeviceProvisioningServicesCertificateProperties
     {
-        /// <summary> Initializes a new instance of CertificateProperties. </summary>
-        public CertificateProperties()
+        /// <summary> Initializes a new instance of DeviceProvisioningServicesCertificateProperties. </summary>
+        public DeviceProvisioningServicesCertificateProperties()
         {
         }
 
-        /// <summary> Initializes a new instance of CertificateProperties. </summary>
+        /// <summary> Initializes a new instance of DeviceProvisioningServicesCertificateProperties. </summary>
         /// <param name="subject"> The certificate&apos;s subject name. </param>
-        /// <param name="expiry"> The certificate&apos;s expiration date and time. </param>
+        /// <param name="expireOn"> The certificate&apos;s expiration date and time. </param>
         /// <param name="thumbprint"> The certificate&apos;s thumbprint. </param>
         /// <param name="isVerified"> Determines whether certificate has been verified. </param>
         /// <param name="certificate"> base-64 representation of X509 certificate .cer file or just .pem file content. </param>
-        /// <param name="created"> The certificate&apos;s creation date and time. </param>
-        /// <param name="updated"> The certificate&apos;s last update date and time. </param>
-        internal CertificateProperties(string subject, DateTimeOffset? expiry, string thumbprint, bool? isVerified, byte[] certificate, DateTimeOffset? created, DateTimeOffset? updated)
+        /// <param name="createdOn"> The certificate&apos;s creation date and time. </param>
+        /// <param name="updatedOn"> The certificate&apos;s last update date and time. </param>
+        internal DeviceProvisioningServicesCertificateProperties(string subject, DateTimeOffset? expireOn, string thumbprint, bool? isVerified, byte[] certificate, DateTimeOffset? createdOn, DateTimeOffset? updatedOn)
         {
             Subject = subject;
-            Expiry = expiry;
+            ExpireOn = expireOn;
             Thumbprint = thumbprint;
             IsVerified = isVerified;
             Certificate = certificate;
-            Created = created;
-            Updated = updated;
+            CreatedOn = createdOn;
+            UpdatedOn = updatedOn;
         }
 
         /// <summary> The certificate&apos;s subject name. </summary>
         public string Subject { get; }
         /// <summary> The certificate&apos;s expiration date and time. </summary>
-        public DateTimeOffset? Expiry { get; }
+        public DateTimeOffset? ExpireOn { get; }
```

Comment:
The three DateTimeOffset properties of this class are marked as "format": "date-time-rfc1123" in swagger, therefore I'm afraid that our deserialization can't work here. Maybe we need to change their type to `string`

Reply:
I think `DataTimeOffset` support rfc1123